pom.xml

```diff
@@ -164,44 +241,21 @@
                 </executions>
             </plugin>
         </plugins>
-        <extensions>
-            <extension>
-                <groupId>org.kuali.maven.wagons</groupId>
-                <artifactId>maven-s3-wagon</artifactId>
-                <version>1.2.1</version>
-            </extension>
-        </extensions>
     </build>
 
     <distributionManagement>
-        <site>
-            <id>s3.site</id>
-            <url>s3://citrine-maven/site</url>
-        </site>
-        <repository>
-            <id>s3.release</id>
-            <url>s3://citrine-maven/release</url>
-        </repository>
         <snapshotRepository>
-            <id>s3.snapshot</id>
-            <url>s3://citrine-maven/snapshot</url>
+            <id>ossrh</id>
+            <url>https://oss.sonatype.org/content/repositories/snapshots</url>
```

Comment:
Will we lose the ability to deploy to our maven repo before it's ready to go to mavn-central? Are we cool with that?